crates/uv-client/src/error.rs

```diff
@@ -61,6 +61,14 @@ impl Error {
                 return true;
             }
 
+            // The server doesn't support rage requests (it doesn't return the necessary headers).
```

Comment:
I think you meant a range request here 😄

Reply:
haha :)